frontend/apps/app/components/SessionDetailPage/components/Output/components/SchemaUpdates/SchemaUpdates.tsx

```diff
@@ -1,22 +1,35 @@
 'use client'
 
+import type { Schema } from '@liam-hq/db-structure'
 import clsx from 'clsx'
 import { MessageSquareCode } from 'lucide-react'
 import { type FC, useState } from 'react'
 import { IconButton } from '@/components'
 import type { ReviewComment } from '@/components/SessionDetailPage/types'
+import { useSchemaUpdates } from './hooks/useSchemaUpdates'
 import { MigrationsViewer } from './MigrationsViewer'
 import styles from './SchemaUpdates.module.css'
 
 type Props = {
-  doc: string
-  comments: ReviewComment[]
+  currentSchema: Schema | null
+  prevSchema: Schema | null
+  comments?: ReviewComment[]
```

Comment:
Is it possible to get rid of these nulls?

Reply:
Thank you very much! It was possible to remove it.🙏
https://github.com/liam-hq/liam/pull/2586/commits/394cb0723cde4620ab154fd0c9bc1af3cd83f74e

Reply:
Thanks!!